.github/workflows/pr-checks.yml

```diff
@@ -39,38 +41,45 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v1
+    - name: 'Checkout Source Code'
+      uses: actions/checkout@v2
+
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v1
 
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+        
     - name: Cache docker layers
-      uses: actions/cache@v1
+      uses: actions/cache@v2
+      id: cache
       with:
-        path: ./caches
-        key: v1-${{ github.head_ref }}
-        
-    - name: Load docker layer cache
-      run: |
-        set +o pipefail
-        docker load -i ./caches/app.tar | true
+        path: /tmp/.buildx-cache
+        key: v2-${{ github.head_ref }}
     
+    - name: 'Docker Prepare'
+      run: docker image prune -a -f
```

Comment:
why do we need to remove other images ? we're missing some spaces ?
if not we could keep them to not slow down

Reply:
@benoitf Had referred to che-dashboard .yml [file](https://github.com/eclipse/che-dashboard/blob/381ff548a9fff3537f1a29ce8e9b228f6c145338/.github/workflows/pr.yaml#L49) for buildx implementation in GA. However, I have removed prune step in #325